docs/dev/onnx_operators.rst

```diff
@@ -697,7 +697,7 @@ Operator Support Matrix
 |                          |           |                 | functions are                |
 |                          |           |                 | not enabled                  |
 +--------------------------+-----------+-----------------+------------------------------+
-| RoiAlign                 | ✅        | FP8, FP16,      |                              |
+| RoiAlign                 | ✅        | FP8, FP16,      |                               |
```

Comment:
The extra whitespace at the end of this row is causing the table to be improperly formatted and not appear on the doc page.

```suggestion
| RoiAlign                 | ✅        | FP8, FP16,      |                              |
```